### PR TITLE
Bug fixes last version adkintunmobile. Not removing manage.py, and ad…

### PR DESCRIPTION
**Pull request by Zeeerooo**
…ded uwsgi to requirements.py

### CODE REVIEW

File: adkintunmobile/server/server/Dockerfile [hunk 1]
```diff
@@ -17,12 +17,11 @@ RUN mkdir /adk && \
...
 	git clone https://github.com/niclabs/AdkintunMobile-Server.git && \
 	cd AdkintunMobile-Server && \
-	git checkout edd2076129f99d1456aa0f47d1c70442844c5a10 
+	git checkout 71ae1535dd513a5698c320863bcb0895382ee286 
 
 # install pip dependencies
 RUN cd /adk/AdkintunMobile-Server && \
-	pip install -r requirements.txt && \
-	pip install uwsgi
+	pip install -r requirements.txt
```
Comment (camilog) [inline adkintunmobile/server/server/Dockerfile:24]: requirements.txt is not changed in this commit?

Comment (Zeeerooo) [reply adkintunmobile/server/server/Dockerfile:24]: yep, is changed in the new adkintunmobile server commit (the new hash added)

Comment (camilog) [reply adkintunmobile/server/server/Dockerfile:24]: Ok!

**Comment by camilog** [comment]
LGTM

**Pull request merged**